sr_vision_mocks/src/sr_vision_mocks/mock_recognize_gazebo_objects.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+#   This node listens to Gazebo's published model states, and publishes a TF
+#   for each object, as well as a RecognizedObjectArray. It is intended to
+#   mock a perfect vision solution, in order to test grasping etc.
+
+import rospy
+import threading
+import tf2_ros
+from gazebo_msgs.msg import ModelStates
+from geometry_msgs.msg import TransformStamped
+from object_recognition_msgs.msg import RecognizedObjectArray, RecognizedObject
+
+
+class MockRecognizeGazeboObjects(object):
+    def __init__(self):
+        self.recognized_object_array = RecognizedObjectArray()
+        self.tfs = []
+        self.lock = threading.Lock()
+        self.br = tf2_ros.TransformBroadcaster()
+        self.r = rospy.Rate(10)
```

Comment:
this is a local variable, no need to use 'self'?

Reply:
No need but no harm, allows use of `r` later in member functions etc.